src/main/java/org/icgc/argo/workflow_management/wes/NextflowService.java

```diff
@@ -267,6 +266,15 @@ private CmdKubeRun createCmd(@NonNull Launcher launcher, @NonNull RunParams para
 
     // Dynamic engine properties/config
     val workflowEngineParams = params.getWorkflowEngineParams();
+    val cluster  =  Objects.nonNull(params.getWorkflowParams().get("cluster"))? params.getWorkflowParams().get("cluster").toString(): "cumulus";
```

Comment:
I would prefer we didn't have a default cluster hardcoded into the application. The workflow application is intended to be run in multiple RDPCs that will not share cluster information. Can we switch this to instead throw an error (UnknownClusterError or something) so that we can move the workflow request to an error state if the config requests a cluster that is not configured in workflow-management?

Reply:
The cluster details come from a config file. Cumulus is the just name used to refer to a set of properties (masterUrl, context and volMounts). The only new property added in this change is the context which is needed by nextflow to access the second cluster. Now, the context property can be left blank but every RDPC will have to provide at least the defaults for volMounts and masterUrl based on how management is designed. 

There is an error handler that handles the scenario where the cluster coming in the request is not configured with management. Here the system logs the issue and puts the workflow run in a SYSTEM_ERROR state.